src/ODESolvers/AdditiveRungeKuttaMethod.jl

```diff
@@ -280,14 +291,15 @@ Giraldo, Kelly and Constantinescu (2013).
 function ARK2GiraldoKellyConstantinescu(F, L,
                                         linearsolver::AbstractLinearSolver,
                                         Q::AT; dt=nothing, t0=0,
-                                        split_nonlinear_linear=false) where {AT<:AbstractArray}
+                                        split_nonlinear_linear=false,
+                                        version=ARK2PaperVersion()) where {AT<:AbstractArray}
```

Comment:
I don't like this version thing: is one of these strictly superior? If not, can we just make it a boolean option (i.e. `paperversion=true`?

Reply:
I made it a boolean option. I would like to do more tests before declaring any of them superior. Picking the paper version as the default for now, since otherwise I would have to update test reference data.